Jenkinsfile

```diff
@@ -490,6 +506,107 @@ az aks create                      \
+                                                    sh "kubectl create clusterrolebinding cluster-system-anonymous --clusterrole cluster-admin --user system:anonymous"
+                                                }
+                                                waitForRollout("kube-system", 30)
+                                            }
+
+                                            // NB: writeJSON doesn't work without approvals(?)
+                                            // See https://issues.jenkins-ci.org/browse/JENKINS-44587
+                                            writeFile([file: 'kubeprod-manifest.jsonnet', text: """
+(import "manifests/platforms/eks.jsonnet") {
+  config:: import "${clusterName}-autogen.json",
+  letsencrypt_environment: "staging",
+  prometheus+: import "tests/testdata/prometheus-crashloop-alerts.jsonnet",
+}
+"""
+                                            ])
+                                        }{
+                                            // update glue records in parent zone
+                                            def nameServers = []
+                                            container('aws') {
+                                                withEnv(["PATH+JQ=${tool 'jq'}"]) {
+                                                    def output = sh(returnStdout: true, script: """
+id=\$(aws route53 list-hosted-zones-by-name --dns-name "${dnsZone}" --max-items 1 | jq -r .HostedZones[0].Id)
+aws route53 get-hosted-zone --id "\$id" --query DelegationSet.NameServers
+"""
+                                                    )
+                                                    for (ns in readJSON(text: output)) {
+                                                        nameServers << ns
+                                                    }
+                                                }
+                                            }
+                                            insertGlueRecords(dnsPrefix, nameServers, "60", parentZone, parentZoneResourceGroup)
+                                        }
+                                    } finally {
+                                        // Delete EKS cluster
+                                        container('eksctl') {
+                                            sh "eksctl delete cluster --name ${clusterName} --timeout 60m"
+                                        }
+                                        container('aws') {
+                                            withEnv([
+                                                "PATH+JQ=${tool 'jq'}",
+                                                "HOME=${env.WORKSPACE}",
+                                            ]) {
+                                                sh """
+CONFIG="${clusterName}-autogen.json"
+ACCOUNT=\$(aws sts get-caller-identity | jq -r .Account)
+DNS_ZONE=\$(cat \${CONFIG} | jq -r .dnsZone)
```

Comment:
Is this just `${dnsZone}`?  If so, we may as well just use the groovy variable directly.

Reply:
Done.